res/bnd/jaspic-api.jar.tmp.bnd

```diff
@@ -35,9 +35,9 @@ Provide-Capability: \
     Implementation-Vendor=Apache Software Foundation
 
 -jpms-module-info: \
-    jakarta.security.enterprise;\
+    jakarta.security.enterprise.api;\
```

Comment:
The most recent RC (2.0.0-RC2) doesn't define a module name so this should be OK for now.